### ISSUE TITLE
Unable to communicate to host service port using alias from docker container

### ISSUE THREAD

**Issue by bikashmishra100** [docker/for-mac#1353]
```
Client:
 Version:      1.13.0
 API version:  1.25
 Go version:   go1.7.3
 Git commit:   49bf474
 Built:        Wed Jan 18 16:20:26 2017
 OS/Arch:      darwin/amd64

Server:
 Version:      1.13.0
 API version:  1.25 (minimum version 1.12)
 Go version:   go1.7.3
 Git commit:   49bf474
 Built:        Wed Jan 18 16:20:26 2017
 OS/Arch:      linux/amd64
 Experimental: true
```
I am using Mac El Capitan 10.11.6. I run a web server on host machine. I am able to access the web server port from inside a running container using the host's ip address. I configured an ip address as an alias using ifconfig command described in documentation https://docs.docker.com/docker-for-mac/networking/. I am able to ping and telnet to the web server using the new alias. But i am unable to ping and telnet from the container to this alias.

### Steps to reproduce the behavior
1. docker run --rm -it busybox
2. sudo ifconfig lo0 alias 10.200.10.1/24
3. ping and telnet 10.200.10.1 from host and docker container.

### Expected behavior
Ping and telnet should work from both host and docker container.

### Actual behavior
Ping and telnet works from host but only ping works from docker container. Telnet to 10.200.10.1 gives a connection refused error.

**Comment by rogaha** [comment]
@bikashmishra100 it should work on the latest `stable/beta` versions. Are you able to upgrade your `Docker for Mac` app?

![screen shot 2017-02-28 at 4 53 26 am](https://cloud.githubusercontent.com/assets/4483394/23406216/bc1fb3c2-fd72-11e6-9916-19ca48209979.png)
![screen shot 2017-02-28 at 4 53 06 am](https://cloud.githubusercontent.com/assets/4483394/23406219/be97a02e-fd72-11e6-8cd1-46ada24ee050.png)

**Comment by rogaha** [comment]
```
rogaha@MacBook-Pro:~/development/rogaha$ sudo ifconfig lo0 alias 10.200.10.1/24                                                            
Password:
rogaha@MacBook-Pro:~/development/rogaha$ docker run -it --rm tutum/curl curl 10.200.10.1                                                                  4:56:33  ☁  master ☀
<!DOCTYPE html>
<html>
<head>
<title>Welcome to nginx!</title>
<style>
    body {
        width: 35em;
        margin: 0 auto;
        font-family: Tahoma, Verdana, Arial, sans-serif;
    }
</style>
</head>
<body>
<h1>Welcome to nginx!</h1>
<p>If you see this page, the nginx web server is successfully installed and
working. Further configuration is required.</p>

<p>For online documentation and support please refer to
<a href="http://nginx.org/">nginx.org</a>.<br/>
Commercial support is available at
<a href="http://nginx.com/">nginx.com</a>.</p>

<p><em>Thank you for using nginx.</em></p>
</body>
</html>
```

**Comment by bikashmishra100** [comment]
@rogaha thank you for the help. I am able to do a telnet after upgrading to the version you have provided in the screenshot. Did i log a duplicate bug?

I need one last help. Without docker0 interface, can i restrict communications from docker container to a fixed address?

**Comment by rogaha** [comment]
@bikashmishra100 I think you can create a `network` interface and assign that to the container. Besides that, the `Docker for Mac` has a fixed IP that you can use, it's `192.168.65.1`. So you don't need to create that alias if you prefer.

**Comment by bikashmishra100** [comment]
@rogaha The fixed IP 192.168.65.1 works nicely. Thanks. Could you please provide sample commands which i can try out for creating network interface and assigning with the container ?

**Comment by rogaha** [comment]
>@rogaha The fixed IP 192.168.65.1 works nicely. 

cool! I'm glad it worked. 
>Could you please provide sample commands which i can try out for creating network interface and assigning with the container ?

sure, here is a link to the docs: https://docs.docker.com/engine/userguide/networking/work-with-networks/#/connect-containers.
